index.html

```diff
@@ -189,13 +190,12 @@ <h3 id="howcanihelp">How can I help?</h3>
 
 end
 </pre>
-        </p>
-      </div>
-
+        <p></p>
+      
       <div class="footer">
-        <pre>&copy; 2013 Michal Siwek</pre>
+        <pre>© 2013-2018 Michal Siwek</pre>
```

Comment:
If we put the date range there, then we will need to change it every year.  Do you want to do that?

Reply:
Yes

Reply:
I can do that